packages/form-validations/README.md

```diff
@@ -98,6 +102,68 @@ const defaultConfig = {
 | pattern   | `pattern="/[a-z]+$/i"` or `data-form-validations-pattern="/[a-z]+$/i"`, `\` must be escaped (replace with `\\`) |                                 |
 | equals    | `data-form-validations-equals="#field-selector"`                                                                | Check that two fields are equal |
 
+## Custom Validator
+
+```javascript
+FormValidations.addValidator(nameOrElem, handler, errorMessage, priority, halt);
+```
+
+### Add a custom validator to a field
+
+```javascript
+var formValidations = new FormValidations(document.getElementById('form1'));
+
+var elem = document.getElementById('email');
+
+// A validator to check if the first letter is capitalized
+formValidations.addElementValidator(
+  elem,
+  function (value) {
+    // here `this` refers to the respective input element
+    if (value.length && value[0] === value[0].toUpperCase()) {
+      return true;
+    }
+    return false;
+  },
+  'The first character must be capitalized',
+  2,
+  false,
+);
+```
+
+### Add a global custom validator
+
+```javascript
+// A validator to check if the input value is within a specified range
+// Global validators must be added before creating the FormValidations instance
+
+FormValidations.addValidator(
+  'my-range',
+  function (value, param1, param2) {
+    // here `this` refers to the respective input element
```

Comment:
there is no this? Or it is only not shown?

Reply:
Removed.